### PR TITLE
always https at the development site

### LINKED ISSUE #328: Downloading nightlys from cgeo.org.development is broken
As stated in the subject line, download links on https://cgeo.org/development don't work, at least not with recent versions of Chrome. Chrome's developer console shows:

```
Mixed Content: The site at 'https://www.cgeo.org/' was loaded over a secure connection, but the file at 'https://download.cgeo.org/cgeo-nightly.apk' was redirected through an insecure connection. This file should be served over HTTPS. This download has been blocked. See https://blog.chromium.org/2020/02/protecting-users-from-insecure.html for more details.
```

The reason for this becomes obvious the page's source code is inspected:
```
<h3 id="download-nightly-builds">Download Nightly Builds</h3>
--
  |  
  | <p><a href="http://download.cgeo.org/cgeo-nightly.apk">c:geo application</a><br />
  | <a href="http://download.cgeo.org/cgeo-nightly-nojit.apk">c:geo application (without JIT)</a><br />
  | <a href="http://download.cgeo.org/cgeo-contacts-nightly.apk">contacts plugin</a></p>
  |  
  | <h2 id="f-droid-repository">F-Droid repository</h2>
```

Since the browser loaded the page via `https`, the download links must also be `https` links otherwise the browser just refuses to follow those links and instead renders the error message shown in the developer console.

Interesting, and at the same time annoying: a regular user who doesn't have the developer console open just sees ***nothing*** except that he doesn't get the desired nightly build.

### PR DESCRIPTION
**Pull request by bekuno**
fixes #328

Change all remaining website links on the development site to https.

### CODE REVIEW

File: development.md [hunk 1]
```diff
@@ -81,9 +81,9 @@ Our repository fingerprint is:
 
 ## Translation of c:geo
 
-If you want to contribute translations, you need a free account at [Crowdin](https://crowdin.com/). Then start contributing new translations to or vote for existing translation proposals in the [c:geo translation project](http://translate.cgeo.org). If you would like to include a new language please contact us via Crowdin.
+If you want to contribute translations, you need a free account at [Crowdin](https://crowdin.com/). Then start contributing new translations to or vote for existing translation proposals in the [c:geo translation project](https://translate.cgeo.org). If you would like to include a new language please contact us via Crowdin.
```
Comment (Lineflyer) [inline development.md:84]: It seems translate.cgeo.org is not working with https.
The CNAME is configured on Crowdin, not sure whats wrong. Anyway we could just link to Crowdins c:geo project page from there.

File: development.md [hunk 2]
```diff
@@ -81,9 +81,9 @@ Our repository fingerprint is:
...
 
-Localization statistics: [![Crowdin](https://badges.crowdin.net/cgeo/localized.svg)](http://translate.cgeo.org)
+Localization statistics: [![Crowdin](https://badges.crowdin.net/cgeo/localized.svg)](https://translate.cgeo.org)
```
Comment (Lineflyer) [inline development.md:86]: see above

File: development.md (hunk 1 above)
Comment (bekuno) [reply development.md:84]: The problem is here:
grafik

I will change the link to crowdin.

Comment (bekuno) [reply development.md]: > I will change the link to crowdin.

Done.

File: development.md (hunk 2 above)
Comment (bekuno) [reply development.md]: Done.

**Pull request merged**